src/transformers/modeling_xlnet.py

```diff
@@ -600,7 +600,7 @@ class XLNetModelOutput(ModelOutput):
 @dataclass
 class XLNetLMHeadModelOutput(ModelOutput):
     """
-    Output type of :class:`~transformers.XLNetModel`.
+    Output type of :class:`~transformers.XLNetLMHeadModel`.
```

Comment:
@sgugger - renamed here. Think this was leftover from copy paste

Reply:
Yes, thanks for fixing!